docs/src/extend/custom-rules.md

```diff
@@ -872,26 +872,28 @@ You can view scope information for any JavaScript code using [Code Explorer](htt
 
 The following table contains a list of AST node types and the scope type that they correspond to. For more information about the scope types, refer to the [`Scope` object documentation](./scope-manager-interface#scope-interface).
 
-| AST Node Type             | Scope Type |
-| :------------------------ | :--------- |
-| `Program`                 | `global`   |
-| `FunctionDeclaration`     | `function` |
-| `FunctionExpression`      | `function` |
-| `ArrowFunctionExpression` | `function` |
-| `ClassDeclaration`        | `class`    |
-| `ClassExpression`         | `class`    |
-| `BlockStatement` ※1       | `block`    |
-| `SwitchStatement` ※1      | `switch`   |
-| `ForStatement` ※2         | `for`      |
-| `ForInStatement` ※2       | `for`      |
-| `ForOfStatement` ※2       | `for`      |
-| `WithStatement`           | `with`     |
-| `CatchClause`             | `catch`    |
-| others                    | ※3         |
+| AST Node Type             | Scope Type           |
+| :------------------------ | :------------------- |
+| `Program`                 | `global`             |
+| `FunctionDeclaration`     | `function`           |
+| `FunctionExpression`      | `function`           |
+| `ArrowFunctionExpression` | `function`           |
+| `ClassDeclaration`        | `class`              |
+| `ClassExpression`         | `class`              |
+| `StaticBlock`             | `class-static-block` |
+| `BlockStatement` ※1       | `block`              |
+| `SwitchStatement` ※1      | `switch`             |
+| `ForStatement` ※2         | `for`                |
+| `ForInStatement` ※2       | `for`                |
+| `ForOfStatement` ※2       | `for`                |
+| `WithStatement`           | `with`               |
+| `CatchClause`             | `catch`              |
+| others                    | ※3 ※4                |
 
 **※1** Only if the configured parser provided the block-scope feature. The default parser provides the block-scope feature if `parserOptions.ecmaVersion` is not less than `6`.<br>
 **※2** Only if the `for` statement defines the iteration variable as a block-scoped variable (E.g., `for (let i = 0;;) {}`).<br>
 **※3** The scope of the closest ancestor node which has own scope. If the closest ancestor node has multiple scopes then it chooses the innermost scope (E.g., the `Program` node has a `global` scope and a `module` scope if `Program#sourceType` is `"module"`. The innermost scope is the `module` scope.).
```

Comment:
```suggestion
**※3** The scope of the closest ancestor node which has own scope. If the closest ancestor node has multiple scopes then it chooses the innermost scope (E.g., the `Program` node has a `global` scope and a `module` scope if `Program#sourceType` is `"module"`. The innermost scope is the `module` scope.).<br>
```

So that `※4` renders on a new line. Currently, it looks like this:

<img width="830" height="218" alt="image" src="https://github.com/user-attachments/assets/7325cea1-9332-4032-93e5-f4f3302ef5d7" />